examples/plugins/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "examples-plugins",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "examples/plugins/src",
+  "projectType": "library",
+  "implicitDependencies": ["cli"],
+  "targets": {
+    "lint": {
+      "executor": "@nx/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["examples/plugins/**/*.ts"]
+      },
+      "dependsOn": ["^build"]
+    },
+    "unit-test": {
+      "executor": "@nx/vite:test",
+      "outputs": ["{options.reportsDirectory}"],
+      "options": {
```

Comment:
Unfortunately, after nx upgrade, the default targets were removed so the vitest `config` option needs to be explicitly added for each test target.